actioncable/lib/action_cable/connection/subscriptions.rb

```diff
@@ -22,7 +22,14 @@ def execute_command(data)
         end
       rescue Exception => e
         @connection.rescue_with_handler(e)
-        logger.error "Could not execute command from (#{data.inspect}) [#{e.class} - #{e.message}]: #{e.backtrace.first(5).join(" | ")}"
+
+        logger.error ""
+        logger.error "\e[31m#{e.class} - #{e.message}\e[0m"
+        logger.error "Request data: \e[34m#{data.inspect}\e[0m"
+
+        e.backtrace.each do |trace_line|
+          logger.error trace_line
+        end
```

Comment:
A variant of this that I played around with was logging scoped to the Rails directory, which cuts down on noise but could hide errors in bundled dependencies:
```rb
project_name = Rails.root.to_s.split("/").last

e.backtrace.each do |trace_line|
  if trace_line.include?(project_name)
    logger.error trace_line
  end
end
```
I assume Rails would rather print out all backlog items so I kept that out of this PR. But that does make this output a bit noisy.